src/test/java/streams/part2/exercise/Exercise3.java

```diff
@@ -0,0 +1,36 @@
+package streams.part2.exercise;
+
+import org.junit.Test;
+
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class Exercise3 {
+
+    @Test
+    public void createLimitedStringWithOddNumbersSeparatedBySpaces() {
+        int countNumbers = 10;
+
+        String result = IntStream
+            .iterate(1, i -> i + 2)
+            .limit(countNumbers)
+            .mapToObj(String::valueOf)
+            .collect(Collectors.joining(" "));
+
+        assertEquals("1 3 5 7 9 11 13 15 17 19", result);
+    }
+
+    @Test
+    public void extractEvenNumberedCharactersToNewString() {
+        String source = "abcdefghijklm";
+
+        String result = source
+            .chars()
+            .filter(i -> (i & 1) == 1)
```

Comment:
I guess, this is a little bit incorrect.
You must take every even char from string.
For example, 
```
| Source     | Expected | Your Result |
| ---------- | -------- | ----------- | 
| 0123456789 | 02468    | 13579       |
| zzzbbcccd  | zzbcd    | ccc         |
```